core/api/src/main/java/org/trellisldp/api/SpecialResources.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.api;
+
+import java.time.Instant;
+import java.util.stream.Stream;
+
+import org.apache.commons.rdf.api.IRI;
+import org.apache.commons.rdf.api.Quad;
+
+/**
+ * Singleton "special" resources.
+ */
+public enum SpecialResources implements Resource {
+
+    MISSING_RESOURCE {
+        @Override
+        public IRI getIdentifier() {
+            return null;
+        }
+
+        @Override
+        public IRI getInteractionModel() {
+            return null;
+        }
+
+        @Override
+        public Instant getModified() {
+            return Instant.EPOCH;
+        }
+
+        @Override
+        public Stream<? extends Quad> stream() {
+            return Stream.empty();
```

Comment:
Should the `default` in `Resource` be `Stream.empty()`?

Reply:
Not a bad idea. In all likelihood, an implementation is still going to need to implement the two `Resource::stream` methods.

Reply:
Actually, given that it's actually hard for me to think of an impl that _wouldn't_ need to override this, better to `default Stream<? extends Quad> stream() { throw new UnimplementedOperationException(); }`.

Reply:
Wait, what am I saying? I need some morning tea. Never mind. Let's leave it unimpled in the interface. I don't know what I'm talking about.

Reply:
Ok, I'll revert that change.